programs/mango-v4/src/state/mango_account.rs

```diff
@@ -629,6 +714,41 @@ impl<
         self.fixed().being_liquidated()
     }
 
+    fn token_conditional_swap_by_index_unchecked(&self, index: usize) -> &TokenConditionalSwap {
+        get_helper(
+            self.dynamic(),
+            self.header().token_conditional_swap_offset(index),
+        )
+    }
+
+    pub fn token_conditional_swap_by_index(&self, index: usize) -> Result<&TokenConditionalSwap> {
+        require_gt!(self.header().token_conditional_swap_count(), index);
+        Ok(self.token_conditional_swap_by_index_unchecked(index))
+    }
+
+    pub fn token_conditional_swap_by_id(&self, id: u64) -> Result<(usize, &TokenConditionalSwap)> {
+        let index = self
+            .all_token_conditional_swap()
+            .position(|tcs| tcs.is_active() && tcs.id == id)
+            .ok_or_else(|| error_msg!("token stop loss with id {} not found", id))?;
```

Comment:
this is the very few place where we directly say "stop loss", nit: maybe reword to token conditional swap with ...?

Reply:
yeah, this is a leftover, will fix